src/sat/monadsat.ml

```diff
@@ -0,0 +1,123 @@
+open Minicat
+
+type clause = int list
+
+type cnf = clause list
+
+let string_of_clause c =
+  "{" ^ String.concat " " (List.map string_of_int c) ^ "}"
+
+let string_of_cnf c =
+  "{" ^ String.concat " " (List.map string_of_clause c) ^ "}"
+
+module MakeSolverMonad(Env : sig type t end) : sig
```

Comment:
The Monad Solver is now in `src/solver/strategy.(ml|i)`. This version called MonadSat is not intended to be used anymore and is incorrect.